Tests/MMEDateTests.m

```diff
@@ -15,8 +15,7 @@ - (void)setUp {
 - (void)testHTTPDateFormatter {
     NSDate *date = [NSDate dateWithTimeIntervalSince1970:214123];
     NSString *dateString = [MMEDate.HTTPDateFormatter stringFromDate:date];
-    
-    XCTAssert([dateString isEqualToString:@"Sat, 3 Jan 1970 03:28:43 -0800"]);
+    XCTAssert([dateString isEqualToString:@"Sat, 3 Jan 1970 03:28:43 -0800"], @"%@", dateString);
```

Comment:
Added a StringFormat comment which includes a formatted comment to inspect the CI box failure which provided the necessary information to discover the timezone was off